kqueen/engines/gce.py

```diff
@@ -189,6 +200,34 @@ def resize(self, node_count, **kwargs):
 
         return True, None
 
+    def set_network_policy(self, network_provider='CALICO', enabled=False):
+
+        unsupported_instances = ['g1-small', 'f1-micro']
+        network_policy_body = {
+            'networkPolicy': {
+                'provider': network_provider,
+                'enabled': enabled
+            }
+        }
+
+        if self.cluster.metadata['node_count'] < 2 or self.cluster_config['cluster']['nodeConfig']['machineType'] in unsupported_instances:
+            msg = 'Setting {} Network Policy for cluster {} denied due unsupported configuration. \
+                   The recommended minimum size cluster to run network policy enforcement is 3 n1-standard-1 instances'.format(network_provider, self.cluster_id)
+            logger.error(msg)
+            return False, msg
+
+        try:
+            self.client.projects().zones().clusters().setNetworkPolicy(projectId=self.project, zone=self.zone, clusterId=self.cluster_id, body=network_policy_body).execute()
```

Comment:
Could you not make such a long lines. For python, max length is 79, but if you want more, at least if should fit the window

Reply:
good point, thx

Reply:
is this json? you have to add json generation function, since in ui would be only text fields or whatever is required